src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -780,7 +780,7 @@ export class CalciteColorPicker {
                   [CSS.splitSection]: true
                 }}
                 dir={elementDir}
-                scale="s"
+                scale={this.scale === "l" ? "m" : "s"}
```

Comment:
nitpick: Seems like `this.scale === "l" ? "m" : "s"` is used a few times. Maybe it could be put into a const?